packages/devtools_app/lib/src/screens/network/har_builder.dart

```diff
@@ -0,0 +1,153 @@
+import '../../../devtools_app.dart';
```

Comment:
This file also needs the copyright header and shouldn't import `devtools_app.dart` directly.

Reply:
done